ydb/core/blobstorage/dsproxy/dsproxy_request.cpp

```diff
@@ -112,6 +112,7 @@ namespace NKikimr {
                         },
                         .NodeLayout = TNodeLayoutInfoPtr(NodeLayoutInfo),
                         .AccelerationParams = GetAccelerationParams(),
+                        .LongRequestThreshold = TDuration::MilliSeconds(LongRequestThresholdMs.Update(TActivationContext::Now())),
```

Comment:
Again, don't use TInstant when measuring time within a single node; it is a subject to sudden jumps when clock gets synced.

Reply:
It's only used to throttle the access to an ICB atomic, it doesn't have to be precise.